samples/user-interface/appwidgets/src/main/java/com/example/platform/ui/appwidgets/glance/weather/WeatherGlanceWidget.kt

```diff
@@ -391,8 +402,8 @@ class UpdateWeatherAction : ActionCallback {
         glanceId: GlanceId,
         parameters: ActionParameters,
     ) {
-        // Force the worker to refresh
-        WeatherWorker.enqueue(context = context, force = true)
+        // Force the widget to refresh
+        WeatherGlanceWidget().updateAll(context)
```

Comment:
You could use the click lambdas now instead of using an action right? It might be simpler